hieradata_aws/class/integration/backend.yaml

```diff
@@ -44,4 +44,6 @@ govuk::apps::local_links_manager::db_hostname: "local-links-manager-postgres"
 govuk::apps::release::db_hostname: "release-mysql"
 govuk::apps::release::db_password: "%{hiera('govuk::node::s_release_db_admin::mysql_db_password')}"
 govuk::apps::service_manual_publisher::db_hostname: "service-manual-publisher-postgres"
+govuk::apps::signon::db_hostname: "signon-mysql"
+govuk::apps::signon::db_password: "%{hiera('govuk::node::s_signon_db_admin::mysql_db_password')}"
```

Comment:
May be that the horse has bolted on this, but any reason why this is in hieradata_aws/class/integration/backend.yaml rather than hieradata_aws/integration.yaml?

Reply:
This only applies the `signon::db_{hostname/password}` values to the `backend` nodes, rather than to _all_ nodes.

Reply:
Don't we want it to all nodes though as per common.yaml?

Reply:
Ah, good spot. I've updated it now 👍